doc/source/indexing.rst

```diff
@@ -667,6 +667,72 @@ For getting *multiple* indexers, using ``.get_indexer``
   dfd.iloc[[0, 2], dfd.columns.get_indexer(['A', 'B'])]
 
 
+.. _indexing.deprecate_loc_reindex_listlike:
+
+Using loc with missing keys in a list is Deprecated
+---------------------------------------------------
+
+.. warning::
+
+   Starting in 0.21.0, using ``.loc`` with missing keys in a list. This is deprecated, in favor of ``.reindex``.
```

Comment:
"using ``.loc`` with missing keys in a list. This is deprecated" -> "using ``.loc`` with a list-like containing missing keys"?